### PR TITLE
fix image size,input field & btn

### LINKED ISSUE #24: bug in the size of the image
### Explanation
it is necessary that the images have the expected size, it is desired that the image size correction be made, so that it does not give the impression that it is in bug.

### Screenshots

Original:
image

Our: 
image

Original:
image

Our: 
image

Original:
image

Our: 
image

Original: 
image

Our: 
image

### Checklist

This issue can be closed when the following tasks are complete:

- [x] GitHub image size is as expected.
- [x] Size of the rectangles that enclose the text are as expected.
- [x] next button size is as expected.
- [x] the size of the copy and go button is as expected.

### References

Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down-width&page-id=0%3A1
Our: https://azordev.github.io/search-by-time-github/

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description

Resize the image size and change input field size,button size

Add some screenshort

imginputbtn

gocopy

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.
#24

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -97,7 +102,9 @@ body {
   text-decoration: none;
   vertical-align: middle;
   cursor: pointer;
-  padding: 0.375rem 0.75rem;
+  /* padding: 0.8rem 0.75rem; */
```
Comment (Israel-Laguan) [inline src/css/style.css]: Is not needed anymore?

Comment (mdrianislam0or1) [reply src/css/style.css]: I think no.

Comment (Israel-Laguan) [reply src/css/style.css]: Ah, please erase the comment then

Comment (mdrianislam0or1) [reply src/css/style.css]: Sorry for this,,Done.

**Comment by e-azocar** [comment]
Please solve conflict

File: src/css/style.css (hunk 1 above)
Comment (Israel-Laguan) [reply src/css/style.css]: Sorry, seems like you forgot to push your changes on this one?

Comment (Israel-Laguan) [reply src/css/style.css]: Please erase the comment that says padding
```suggestion
```

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Pull request merged**